packages/redux/src/orders/actions/factories/helpers/normalizeFetchOrderResponse.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
This is to be removed, right?

Reply:
Yes, however, we will not be making this change on the current PR. We intend to consolidate multiple stories associated with this initiative into this branch (`feat_split_orders`) prior to merging with the `next` branch once the entire initiative has been completed.

Reply:
Understood, thanks!